dask/dataframe/core.py

```diff
@@ -1348,6 +1348,12 @@ def to_csv(self, filename, **kwargs):
 
         return to_csv(self, filename, **kwargs)
 
+    def to_sql(self, *args, **kwargs):
+        """ See dd.to_sql docstring for more information """
```

Comment:
Isn't this `dd.to_sql`? Perhaps you mean pandas.

Reply:
I'd prefer to use `@derived_from(pd.DataFrame)`. Pandas doesn't have a top-level `to_sql` method.

Reply:
The top-level `dd.to_csv`, etc. predate `derived_from`. It's fine to still have a `dd.to_sql`, but the docstring should be on DataFrame.to_sql.